### ISSUE TITLE
Add support for the mirrors role

### ISSUE THREAD

**Issue by heartsucker** [heartsucker/rust-tuf#4]

**Comment by trishankkarthik** [comment]
Fortunately, this has so far been an optional, and the least important role in TUF :) I think [the map file](https://github.com/theupdateframework/taps/blob/JustinCappos-patch-2/tap4.md) should be able to completely replace this role... @JustinCappos

**Comment by heartsucker** [comment]
@trishankkarthik Yes, this is true. Someone I know in SF wanted to help out, so I started making some basic tickets that for now would let others contribute without me stepping all over them.

**Comment by trishankkarthik** [comment]
Thanks for all your efforts! Looking forward...
